builds/e2e/nested-e2e.yaml

```diff
@@ -107,25 +107,50 @@ stages:
           IotHubConnectionString: $(ConnectivityNestedEdge-ConnectionString)
 
   - job:  Clean_up     
-    dependsOn: SetupVM_and_RunTest_level3
+    dependsOn:
+      - SetupVM_level5 
+      - SetupVM_level4
+      - SetupVM_and_RunTest_level3
     condition: always()
     displayName: Clean up identities and unlock agents
     timeoutInMinutes: 1    
     variables:    
-      deviceLvl5Name: $[ dependencies.SetupVM_level5.outputs['deployIoTEdge.deviceName'] ]
-      deviceLvl5ConnectionString: $[ dependencies.SetupVM_level5.outputs['createIdentity.connectionString'] ]     
-      deviceLvl4Name: $[ dependencies.SetupVM_level4.outputs['deployIoTEdge.deviceName'] ]
-      deviceLvl4ConnectionString: $[ dependencies.SetupVM_level4.outputs['createIdentity.connectionString'] ] 
+      deviceLvl5DeviceId: $[ dependencies.SetupVM_level5.outputs['createIdentity.parentDeviceId'] ] 
+      deviceLvl4DeviceId: $[ dependencies.SetupVM_level4.outputs['createIdentity.parentDeviceId'] ] 
+      deviceLvl5IoTHubName: $[ dependencies.SetupVM_level5.outputs['createIdentity.iotHubName'] ] 
```

Comment:
Do we have use case for two different hubs?

Reply:
no we don't. Out of scope for now

Reply:
So we won't need 2 variables, right? And hub name shouldn't require get back from job but load from config or key vault.